docs/verbs/catkin_build.rst

```diff
@@ -533,6 +533,15 @@ So if you want to check for failing tests, use the ``catkin_test_results`` comma
 
 The result code will be non-zero unless all tests passed.
 
+Building With Warnings
+^^^^^^^^^^^^^^^^^^^^^^
+
+It can sometimes be useful to compile with strict warnings enabled across your whole catkin workspace. To achieve this, use a command like:
+
+.. code-block:: bash
+
+    $ catkin build -v --cmake-args -DEXTRA_C_FLAGS="-Wall -g -W -Wno-unused-parameter"
```

Comment:
I don't think I've ever seen `EXTRA_C_FLAGS`, is that a CMake thing? I've always used `CMAKE_C_FLAGS` or `CMAKE_CXX_FLAGS`, do you know if `EXTRA_C_FLAGS` is documented anywhere?